docs/install.md

```diff
@@ -12,50 +12,63 @@ weight=4
 
 # Install Docker Compose
 
-To install Compose, you'll need to install Docker first. You'll then install
-Compose with a `curl` command. 
+You can run Compose on OS X and 64-bit Linux.  It is currently not supported on
+the Windows operating system. To install Compose, you'll need to install Docker
+first. 
 
-## Install Docker
+Depending on how your system is configured, you may require `sudo` access to
+install Compose. If your system requires `sudo`, you will receive "Permission
+denied" errors when installing Compose. If this is the case for you, preface the
+install commands with `sudo` to install.
 
-First, install Docker version 1.7.1 or greater:
+To install Compose, do the following:
 
-- [Instructions for Mac OS X](http://docs.docker.com/installation/mac/)
-- [Instructions for Ubuntu](http://docs.docker.com/installation/ubuntulinux/)
-- [Instructions for other systems](http://docs.docker.com/installation/)
+1. Install Docker version 1.7.1 or greater:
 
-## Install Compose
+   - <a href="https://docs.docker.com/installation/mac/" target="_blank">Mac OS X installation</a>
+   - <a href="https://docs.docker.com/installation/ubuntulinux/" target="_blank">Ubuntu installation</a>
+   - <a href="https://docs.docker.com/installation/" target="_blank">other system installations</a>
+  
+     The Mac OS X installation using Docker Toolbox also installs Compose for you.
```

Comment:
I fear lots of users might skip over this message and continue on to step 2. Instead of a step-by-step guide with a break point, I wonder if this would be better structured as two sections:

1) If you're on OS X, install Toolbox.

2) If you're on Linux, or want to install manually for whatever reason, do this:
a) ..
b) ..